test/parallel/test-util-internal.js

```diff
@@ -8,9 +8,24 @@ const fixtures = require('../common/fixtures');
 const {
   getHiddenValue,
   setHiddenValue,
-  arrow_message_private_symbol: kArrowMessagePrivateSymbolIndex
+  arrow_message_private_symbol: kArrowMessagePrivateSymbolIndex,
+  safeGetenvForJs
 } = process.binding('util');
 
+for (const oneEnv in process.env) {
+  console.log(oneEnv, safeGetenvForJs(oneEnv));
+  assert.strictEqual(
+    safeGetenvForJs(oneEnv),
+    process.env[oneEnv]
+  );
+}
+
+const FAKEENVVAR = 'FAKEYFAKEENVVAR';
+assert.strictEqual(
+  safeGetenvForJs(FAKEENVVAR),
+  undefined
+);
```

Comment:
I guess it's unlikely that anyone has this set in the environment but it's less robust than I'd like.

Reply:
I'll remove it then.